components/kyma-environment-broker/internal/process/provisioning/audit_log_step.go

```diff
@@ -118,6 +114,16 @@ func (alo *AuditLogOverrides) Run(operation internal.ProvisioningOperation, logg
 	return operation, 0, nil
 }
 
+func (alo *AuditLogOverrides) injectOverrides(aloOv auditlog.Overrides, tmp *template.Template, log logrus.FieldLogger) (string, error) {
+	var flOutputs bytes.Buffer
+	err := tmp.Execute(&flOutputs, aloOv)
+	if err != nil {
+		log.Errorf("Template error while injecting cls overrides: %v", err)
+		return "", err
```

Comment:
Let's just wrap the error here instead of logging it in the previous line.

Reply:
done.